backend/src/models/answer.js

```diff
@@ -8,20 +6,11 @@ class Answer {
     }
 
     /* TODO change this */
-    postAnswer(questionID, { body }) {
-        const self = this
-        return new Promise((resolve, reject) => {
-            const columns = ['userID', 'questionID', 'body']
-            const placeholders = columns.map(_ => '?').join()
-            const query = `INSERT INTO answer (${columns}) VALUES (${placeholders})`
-            this.db._db.run(
-                query,
-                [0, questionID, body], // TODO user id is a placeholder obviously, but it can't be null so...
-                // TODO meaningful error message/code based on sql error
-                // TODO proper error handling for all the endpoints :/
-                function (err) { err ? reject(Error('Error adding answer')) : resolve(self.getAnswers(questionID, 1)) }
-            )
-        })
+    postAnswer(questionID, { body, userID = 1 }) {
+        return this.db
+            .insert('answer', { questionID, body, userID })
+            /* Still not sure on this, seems wasteful to send all new data */
```

Comment:
I guess the alternative is selecting the answer we just inserted into the table and returning that. I think it's not worth the effort of adding right now though.

Reply:
Only thing I would say is that postQuestion and postAnswer currently behave differenlty.

And all the relevant data _except_ the id is already client side.

Reply:
yeah i spose

Reply:
But, look I'll leave it for now.